pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -59,7 +59,8 @@ def configure_checkpoint_callbacks(self, checkpoint_callback: Union[ModelCheckpo
             # TODO: deprecated, remove this block in v1.3.0
             rank_zero_warn(
                 "Passing a ModelCheckpoint instance to Trainer(checkpoint_callbacks=...)"
-                " is deprecated since v1.1 and will no longer be supported in v1.3.",
+                " is deprecated since v1.1 and will no longer be supported in v1.3."
+                "Use `callbacks` argument instead.",
```

Comment:
```suggestion
                " Use `callbacks` argument instead.",
```
needs the space after period.